core-bundle/src/Resources/contao/library/Contao/Date.php

```diff
@@ -638,6 +638,12 @@ protected static function resolveCustomModifiers($strDate)
 
 		foreach ($chunks as $chunk)
 		{
+			if (strpos($chunk, '::') === false)
+			{
+				$strReturn .= $chunk;
+				continue;
+			}
+
 			list($index, $flag) = explode('::', $chunk);
```

Comment:
```suggestion
			$chunks = explode('::', $chunk);
			$index = $chunks[0];
			$flag = $chunks[1] ?? null;
```

Reply:
I fixed this earlier using `array_pad`, which makes explicit what you did manually. But I guess both are fine.